source/plugins/go/input/lib/clusteridentity.go

```diff
@@ -0,0 +1,299 @@
+package lib
+
```

Comment:
I thought this file was used for custom metrics? Is that not the case? why do we use this?